modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/OrganizationResourceDTOConverter.java

```diff
@@ -131,11 +173,13 @@ public Organization toDTO(
 					});
 				setImageId(organization::getLogoId);
 				setKeywords(
-					() -> ListUtil.toArray(
-						_assetTagLocalService.getTags(
-							organization.getModelClassName(),
-							organization.getOrganizationId()),
-						AssetTag.NAME_ACCESSOR));
+					() -> NestedFieldsSupplier.supply(
+						"keywords",
+						nestedFieldNames -> ListUtil.toArray(
+							_assetTagLocalService.getTags(
+								organization.getModelClassName(),
+								organization.getOrganizationId()),
+							AssetTag.NAME_ACCESSOR)));
```

Comment:
because this already exists please don't change it putting it under nested field, otherwise it will be a breaking change

Reply:
and because we don't change it we can remove it from the test

Reply:
Ok